src/xcode/ENA/ENA/Source/Services/__tests__/ExposureSubmissionServiceTests.swift

```diff
@@ -401,7 +401,8 @@ class ExposureSubmissionServiceTests: XCTestCase {
 		let client = ClientMock()
 		store.registrationToken = "dummyRegistrationToken"
 
-		let testResult = TestResult.allCases.randomElement() ?? TestResult.positive
+		let storedResults: [TestResult] = [.pending, .negative, .positive, .invalid]
```

Comment:
So shouldn't we then create a test to expect an error response in case of .expired results?

Reply:
Yes, maybe - I only fixed the behavior I found

Reply:
I have included all other cases in tests to make it more reliable